examples/gno.land/r/gov/dao/v3/README.md

```diff
@@ -0,0 +1,185 @@
+# GOVDAO Specifications
+
+## 1. Overview
+**GovDAO** is a governance body with three membership tiers—**T1**, **T2**, and **T3**—along with defined voting power, membership requirements, size targets, and compensation policies. 
+
+---
+
+## 2. Membership Tiers
+
+### 2.1 Tier Definitions
+- [X] **T1 (Core Tier)**  
+  - [X] Highest tier; self-selecting membership with a *supermajority* vote from T1.
+  - [X] Membership can only be withdrawn by supermajority vote *with cause*.
+- [X] **T2**  
+  - [X] Selected by GovDAO with T3 abstaining, requiring a *simple majority* vote.
+  - [X] Membership can be withdrawn for any reason.
+- [X] **T3 (General Tier)**  
+  - [X] Lowest tier; *permissionless invitation* from T1 and T2 members.
```

Comment:
What is the meaning of `permissionless` in this context? T1, T2 and T3 all use their invitation points so what's the difference between T1/T2 and T3?

Reply:
Ok got it, thanks 👍